cookbooks/shell_simple_2d_smoothing/doc/shell_simple_2d_smoothing.md

```diff
@@ -16,45 +17,32 @@ Instead of just looking at an individual cell&rsquo;s artificial viscosity,
 improvements in the minimizing of the oscillations can be made by smoothing.
 Smoothing is the act of finding the maximum artificial viscosity taken over a
 cell $T$ and the neighboring cells across the faces of $T$, i.e.,
-$$\bar{v_h}(T) = \max_{K \in N(T)} v_h(K)$$ where $N(T)$ is the set containing
-$T$ and the neighbors across the faces of $T$.
+```{math}
+\bar{v_h}(T) = \max_{K \in N(T)} v_h(K)
+```
+where $N(T)$ is the set containing $T$ and the neighbors across the faces of $T$.
 
-This feature can be turned on by setting the [Use artificial viscosity
-smoothing][] flag inside the [Stabilization][] subsection inside the
-[Discretization][] subsection in your parameter file.
+This feature can be turned on by setting the `Use artificial viscosity smoothing`
+flag inside the `Stabilization` subsection inside the `Discretization`
+subsection in your parameter file.
 
 To show how this can be used in practice, let us consider the simple
 convection in a quarter of a 2d annulus cookbook in Section
-[\[sec:shell-simple-2d\]][1], a radial compositional field was added to help
+{ref}`sec:shell-simple-2d`, a radial compositional field was added to help
 show the advantages of using the artificial viscosity smoothing feature.
 
 By applying the following changes shown below to the parameters of the already
 existing file
 
-    cookbooks/shell_simple_2d/shell_simple_2d.prm,
-
-``` prmfile
+```{literalinclude} cookbooks/shell_simple_2d/shell_simple_2d.prm
 ```
 
 it is possible to produce pictures of the simple convection in a quarter of a
-2d annulus such as the ones visualized in Figure&nbsp;[\[fig:smoothing\]][2].
-
-&nbsp;
-
-<div id="refs" class="references csl-bib-body hanging-indent">
-
-<div id="ref-guer11" class="csl-entry">
-
-Jean-Luc Guermond and Richard Pasquetti and Bojan Popov. 2011. &ldquo;Entropy
-Viscosity Method for Nonlinear Conservation Laws.&rdquo; *Journal of
-Computational Physics* 230: 4248&ndash;67.
-
-</div>
+2d annulus such as the ones visualized in Figure&nbsp;{ref}`fig:smoothing`.
 
-</div>
+```{figure-md} fig:smoothing
+<img src="artificial_viscosity_smoothing.*" style="width:100.0%" />
```

Comment:
Somehow this image does not render in readthedocs, take a look at https://aspect-documentation--4718.org.readthedocs.build/en/4718/user/cookbooks/cookbooks/shell_simple_2d_smoothing/doc/shell_simple_2d_smoothing.html